special-interest-groups/sig-template/member-list.md

```diff
@@ -0,0 +1,59 @@
+# Template SIG Member List
+
+**NOTE**:
+
+* This member list is updated on a bi-weekly basis.
+* Last update time: yyyy-mm-dd
+
+## Tech Leads
+
+* [Tech lead 1](Github-link-of-tech-lead-1), [Organization](link-of-organization)
+* [Tech lead 2](Github-link-of-tech-lead-2), [Organization](link-of-organization)
+* [Tech lead 3](Github-link-of-tech-lead-3), [Organization](link-of-organization)
+
+
+## Committers
+
+Members and their contributions:
+
+| user                                              | num merged PRs | earliest PR merged time | latest PR merged time |
```

Comment:
- How about capitalize the first letter in each column? Such as `user` > `User`
- Suggest deleting extra unnecessary spaces to make this table more friendly when editing
```suggestion
| User | Num of merged PRs | First PR merged time | Last PR merged time |
```